src/client/containers/OrderConfirmationPage/OrderConfirmationPage.styles.css

```diff
@@ -1,3 +1,57 @@
-div {
-  color: #687808;
+.confirmation-page {
+    align-items: center;
+    margin: 4%;
 }
+
+.confirmation-page .inner-div {
```

Comment:
How about `page-content` instead?
Or if you're applying something like [BEM](http://getbem.com/naming/), how about `.confirmation-page__content`? 
Seeing as all of your class named are scoped to the `confirmation-page` class this might be a better solution.

Reply:
Okay .. I will update to page-content